documentation/modules/overview/con-configuration-points-broker.adoc

```diff
@@ -9,57 +9,58 @@ A kafka cluster comprises one or more brokers.
 For producers and consumers to be able to access topics within the brokers, Kafka configuration must define how data is stored in the cluster, and how the data is accessed.
 You can configure a Kafka cluster to run with multiple broker nodes across racks.
 
-Storage::
-Kafka and ZooKeeper store data on disks.
-+
-Strimzi requires block storage provisioned through `StorageClass`.
-The file system format for storage must be _XFS_ or _EXT4_.
-Three types of data storage are supported:
-+
---
+== Storage
+Strimzi supports the following storage configuration options for managing Kafka and ZooKeeper data:
+
 Ephemeral (Recommended for development only):: Ephemeral storage stores data for the lifetime of an instance. Data is lost when the instance is restarted.
 Persistent:: Persistent storage relates to long-term data storage independent of the lifecycle of the instance.
 JBOD (Just a Bunch of Disks, suitable for Kafka only):: JBOD allows you to use multiple disks to store commit logs in each broker.
---
-+
+Tiered storage (Early access):: Tiered storage is an early access feature in Kafka. Tiered storage provides more flexibility for data management by leveraging the parallel use of storage types with different characteristics.
+
+Storage types are specified in the custom resource using `storage` or `tieredStorage` configuration properties:
+
+* `storage.type: ephemeral`
+* `storage.type: persistent-claim` for persistent storage using {K8sPersistentVolumeClaims} (PVCs)
+* `storage.type:jbod` for JBOD storage
+* `tieredStorage.type:custom` for custom tiered storage
```

Comment:
I think the highlighted code is confusing. I would use something like this ...

```
* `type: persistent-claim` in the `storage` object for persistent storage using {K8sPersistentVolumeClaims} (PVCs)
```

Reply:
I've grouped by storage properties:

-  `storage` configuration types:
    - `type: ephemeral`
    - `type: persistent-claim` for persistent storage using {K8sPersistentVolumeClaims} (PVCs)
    - `type:jbod` for JBOD storage
- `tieredStorage` configuration types:
    - `type:custom` for custom tiered storage